src/resources/authoring/gv2.json

```diff
@@ -1019,13 +1023,26 @@
               {
                 "name": "Challenge 1.1.1",
                 "id": "allele-5drakes-tutorial",
-                "room": "simroom"
+                "room": "simroom",
+                "dialog": [
+                  {
+                    "character": "YEUNG",
+                    "text": "Good to meet you, Cadet! I'm Professor Yeung, and I'm in charge of the Sim Room. We’re going to show you on this simulator how to breed five different kinds of drakes - with different combinations of arms, legs, wings, horns, and armor."
```

Comment:
From our voice meeting, thinking that something like the whole authoring file being localized may possibly make more sense.